frontend/src/utils/chat/markdown.js

```diff
@@ -5,7 +5,7 @@ import "highlight.js/styles/github-dark-dimmed.min.css";
 import { v4 } from "uuid";
 
 const markdown = markdownIt({
-  html: false,
+  html: true,
```

Comment:
Unfortunately, we cannot enable this as it is _technically_ unsafe.
https://github.com/markdown-it/markdown-it/blob/master/docs/security.md#security

The reason is was changed a while back was because of an odd issue during streaming where if you are streaming an input and under the proper circumstances the chunks come in with a `<script>alert('XSS');</script>` it would execute that script instead of rendering it. This also can occur with `img` and other tags. 

So while this _does work_ for your specific issue we cannot enable it this way as it will retroactively undo a fix for another bug

Reply:
Oh, I apologize for not noticing that such a case would occur with that setting. Thank you very much for resolving that issue for me.

Reply:
I didnt think about it either until is happened to someone asking a question about XSS and then it happened right in their browser haha